### PR TITLE
showpassword

### PR DESCRIPTION
**Pull request by Harshpawar333**
password button
reset button
confirm password

### CODE REVIEW

File: src/login.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import logo from './logo.svg'
+import 'bootstrap/dist/css/bootstrap.min.css'
+import './App.css'
+
+import { useState } from 'react'
```
Comment (visha12l) [inline src/login.js:5]: this should be first line in your file

File: src/login.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  const [passwordInput, setPasswordInput] = useState('password')
+  const [confirmPasswordErr, setConfirmPasswordErr] = useState(false)
+  function handleLogin(){
+    const mailformat = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/
+    if (email.match(mailformat)) {
+        setEmailErr(false)
+      } else {
+        setEmailErr(true)
+      }
+      if (password.length < 8) {
```
Comment (visha12l) [inline src/login.js:21]: remove password.length condition add password === ""

File: src/login.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        setPasswordErr(true)
+      } else {
+        setPasswordErr(false)
+      }
+  }
+  return (
+    <body>
```
Comment (visha12l) [inline src/login.js:28]: remove body tag from here

File: src/login.js [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+                />
+              </a>
+            </div>
+          </nav>
+          <div class="py-4 pt-0 row">
+            <div class="col-sm-9">
+              <input
+                type="text"
+                minlength="8"
+                required
```
Comment (visha12l) [inline src/login.js:48]: remove this line

File: src/login.js [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+                class="form-control"
+                placeholder="Email Addresss or Phone Number"
```
Comment (visha12l) [inline src/login.js:50]: placeholder="Email Address"

File: src/test.txt [hunk 6]
```diff
@@ -47,4 +47,20 @@ Add show password button
...
\ No newline at end of file
+reset- to reset all data
+
+
+login changes
+data types
+email  => string
+password => string
+
+password => password length should be greater than 8
```
Comment (visha12l) [inline src/test.txt:58]: password => password should not be empty

File: src/login.js (hunk 3 above)
Comment (Harshpawar333) [reply src/login.js:26]: css applied in bodytag for padding

**Review by visha12l — APPROVED** [review]

**Pull request merged**